creating_images/guidelines.adoc

```diff
@@ -124,7 +119,7 @@ them, where possible, so they do not end up written to a layer.
 
 In addition, performing multiple commands in a single `RUN` statement reduces
 the number of layers in your image, which improves download and extraction time.
-
+endif::[]
```

Comment:
I don't see any reason for this to be hidden for online. These are part of the recommendations on how to build images to use in OpenShift. This section does not describe using the docker build strategy in Openshift.

@sferich888 would you like to confirm?